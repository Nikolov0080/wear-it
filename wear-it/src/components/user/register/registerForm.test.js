import React from 'react';
import renderer from 'react-test-renderer';
import RegisterForm from './registerForm';
import { BrowserRouter } from 'react-router-dom';

it('renders correctly Register', () => {
    const tree = renderer
        .create(<BrowserRouter><RegisterForm /></BrowserRouter>)
        .toJSON();
    expect(tree).toMatchSnapshot();
});