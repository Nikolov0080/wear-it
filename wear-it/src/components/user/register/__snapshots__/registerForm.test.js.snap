// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly Register 1`] = `
<div
  className="box"
>
  <div
    className="registerForm"
    onChange={[Function]}
  >
    <h2
      className="title"
    >
      REGISTER
    </h2>
    
    <form
      onSubmit={[Function]}
    >
      <div>
        <input
          autoComplete="off"
          className="input"
          name="username"
          onChange={[Function]}
          placeholder="Username"
          type="text"
        />
      </div>
      <div>
        <input
          autoComplete="off"
          className="input"
          name="email"
          onChange={[Function]}
          placeholder="Email"
          type="email"
        />
      </div>
      <div>
        <input
          autoComplete="off"
          className="input"
          name="password"
          onChange={[Function]}
          placeholder="Password"
          type="password"
        />
      </div>
      <div>
        <input
          autoComplete="off"
          className="input"
          name="rePassword"
          onChange={[Function]}
          placeholder="Repeat Password"
          type="password"
        />
      </div>
      <div>
        <button
          className="btn"
          type="submit"
        >
          submit
        </button>
      </div>
    </form>
  </div>
</div>
`;
